code/modules/mob/living/carbon/human/human.dm

```diff
@@ -9,6 +9,7 @@
 	var/list/hud_list[9]
 	var/datum/species/species //Contains icon generation and language information, set during New().
 	var/embedded_flag	  //To check if we've need to roll for damage on movement while an item is imbedded in us.
+	var/percent_machine = 0
```

Comment:
Does that add the var to the specific human?
I only did this because I want it to be a semi-permanent thing on the specific person.